rootconf/armv8a/sysroot-bin/onie-boot-mode

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```

Comment:
same comment -- can these aarch64 rootconfig files be combined with x86_64 and called "grub-arch" ?